docs/run-integration-tests.md

```diff
@@ -0,0 +1,34 @@
+# Run integration tests
+
+## Writing tests and running them locally
+
+1. Create `.env` file at the root of the repository with the following variables:
+
+```bash
+CONTENTFUL_CMA_TOKEN=<personal-cma-token>
+CONTENTFUL_SPACE=<testing-space-id>
+CYPRESS_BASE_URL=https://app.contentful.com
```

Comment:
I'd name it `WEB_APP_BASE_URL`

Reply:
`CYPRESS_BASE_URL` is a required env name for overriding `baseUrl`